module/VuFind/src/VuFind/ILS/Driver/DAIA.php

```diff
@@ -885,54 +931,82 @@ protected function getItemStatus($item)
                         );
                         $item_limitation_types = array_merge(
                             $item_limitation_types,
-                            $this->getItemLimitationTypes($unavailable['limitation'])
+                            $this->getItemLimitationTypes(
+                                $unavailable['limitation']
+                            )
                         );
                     }
-                }
-                // attribute expected is mandatory for unavailable element
-                if (!empty($unavailable['expected'])) {
-                    try {
-                        $duedate = $this->dateConverter
-                            ->convertToDisplayDate(
-                                'Y-m-d',
-                                $unavailable['expected']
+                    // attribute expected is mandatory for unavailable
+                    // element
+                    if (!empty($unavailable['expected'])) {
+                        try {
+                            $duedate = $this->dateConverter
+                                ->convertToDisplayDate(
+                                    'Y-m-d',
+                                    $unavailable['expected']
+                                );
+                        } catch (\Exception $e) {
+                            $this->debug(
+                                'Date conversion failed: ' . $e->getMessage()
                             );
-                    } catch (\Exception $e) {
-                        $this->debug('Date conversion failed: ' . $e->getMessage());
-                        $duedate = null;
+                            $duedate = null;
+                        }
                     }
-                }
 
-                // attribute queue can be set
-                if (isset($unavailable['queue'])) {
-                    $queue = $unavailable['queue'];
-                }
+                    // attribute queue can be set
+                    if (isset($unavailable['queue'])) {
+                        $queue = $unavailable['queue'];
+                    }
 
-                // log messages for debugging
-                if (isset($unavailable['message'])) {
-                    $this->logMessages($unavailable['message'], 'item->unavailable');
+                    // log messages for debugging
+                    if (isset($unavailable['message'])) {
+                        $this->logMessages(
+                            $unavailable['message'],
+                            'item->unavailable'
+                        );
+                    }
                 }
             }
         }
 
-        /*'returnDate' => '', // false if not recently returned(?)*/
+        // gather more information about the provided service
+        if (isset($item['about'])) {
+            $item_notes[] = $item['about'];
+        }
+
+        // If no useful availability service is found, return only the
```

Comment:
The added logic block below will introduce very strict requirements for DAIA items in terms of availablity modelling: according to this if-conditional the unavailability information of an unavailable item will be shown to the user only if either a link, a duedate or an available service exists. Otherwise this item will simply be shown as unavailable. This is too strict as according to DAIA specification (http://gbv.github.io/daia/daia.html#availability) the only required field for defining an unavailable service is `service`. E.g. the availability of an unavailable item could be further specified by a given limitation, but any limitation will be ignored if the above mentioned conditions are not met.